src/timesync/timesyncd-server.c

```diff
@@ -60,6 +60,20 @@ ServerAddress* server_address_free(ServerAddress *a) {
         return mfree(a);
 }
 
+static int enable_ntp_server_defer_event(Manager *m, ServerType type) {
+        assert(m);
+        int r;
+
```

Comment:
assert that server type is >=0 and < _SERVER_TYPE_MAX

Reply:
done